plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestingHadoopPlugin.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.spi.Plugin;
+import io.trino.spi.connector.ConnectorFactory;
+
+public class TestingHadoopPlugin
```

Comment:
Maybe we could use existing ```io.trino.plugin.hive.TestingHivePlugin``` but make metastore instance optional in it ?

Reply:
Same as above reply.

Reply:
NIT: Providing metastore may be optional. If none provided we use normal module setup, but we will have single configurable testing plugin.

Reply:
> Same as above reply.

which one?

Reply:
`TestingHadoopPlugin` no longer exists.